bibernate-core/src/main/java/com/bobocode/petros/bibernate/annotations/Table.java

```diff
@@ -13,5 +13,4 @@
     String name();
 
     String schema() default "";
-
```

Comment:
Try to avoid such modifications to the classes where you haven't made any changes